components/OnboardingModal/OnboardingModal.tsx

```diff
@@ -29,18 +31,30 @@ export default function OnboardingModal({
 
   const handleAdvance = () => {
     if (isLastSlide) return;
-    setCurrentSlide((prev) => prev + 1);
+    setCurrentSlide(prev => prev + 1);
+  };
```

Comment:
this seems like extra code where it could be `simpler`

```
if (isLastSlide) {
router.push(`/${wagmiAddress}`);
return 
} 
setCurrentSlide(prev => prev + 1);
```

If you make that update, you can remove all the other code changes you have on this page, right?